base/applications/regedit/clb/clb.c

```diff
@@ -1,29 +1,8 @@
 /*
- * ReactOS Column List Box
- * Copyright (C) 2005 Thomas Weidenmueller
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
- */
-/*
- * PROJECT:         ReactOS Column List Box
- * FILE:            base/applications/regedit/clb/clb.c
- * PURPOSE:         Column List Box
- * PROGRAMMER:      Thomas Weidenmueller <w3seek@reactos.com>
- *
- * UPDATE HISTORY:
- *      10/29/2005  Created
+ * PROJECT:    ReactOS Column List Box
+ * LICENSE:    LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:    Column List Box
+ * PROGRAMMER: 2005 Thomas Weidenmueller <w3seek@reactos.com>
```

Comment:
For all the files in this PR where you made this change, please use the correct file header format:
https://reactos.org/wiki/Coding_Style#File_Structure

Reply:
Resolved, see https://github.com/reactos/reactos/pull/7193/commits/2bbc10138a320054b1327ea05ab0756c447fd4e9

Reply:
thanks